docs/usage.md

```diff
@@ -37,6 +41,18 @@ license.meta["permissions"]
 => ["commercial-use","modifications","distribution","private-use"]
 ```
 
+If you wish to scan private Github repositories, you can configure the embedded [Octokit](https://github.com/octokit/octokit.rb) client
+using the standard module-level configuration, for example:
+
+```ruby
+# see https://github.com/octokit/octokit.rb#configuring-module-defaults
+Octokit.configure do |c|
+  c.access_token = "<your 40 char token>"
+end
+
+license = Licensee.license "https://github.com/benbalter/licensee"
+```
+
```

Comment:
It may also be worth noting, that for command-line invocations, you can pass it as an environmental variable, e.g.:

```
OCTOKIT_ACCESS_TOKEN=abc123 licensee foo/bar
```